### PR TITLE
added the bucket inside the resources

### PR DESCRIPTION
**Pull request by murad-ali-MoJ**
added mojap-manage-offences and mojap-hub-exports buckets resources

### CODE REVIEW

File: terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf [hunk 1]
```diff
@@ -14,6 +14,10 @@ data "aws_iam_policy_document" "create_a_derived_table" {
       "arn:aws:s3:::mojap-derived-tables",
       "arn:aws:s3:::dbt-query-dump/*",
       "arn:aws:s3:::dbt-query-dump",
+      "arn:aws:s3:::mojap-manage-offences/*",
+      "arn:aws:s3:::mojap-manage-offences",
+      "arn:aws:s3:::mojap-hub-exports/*",
```
Comment (jhpyke) [inline terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf]: Hi - Hub exports is a bucket used by a lot of sensitive stuff - this is too broad. Can you rescope to the actual folder used

File: terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf [hunk 2]
```diff
@@ -14,6 +14,10 @@ data "aws_iam_policy_document" "create_a_derived_table" {
...
+      "arn:aws:s3:::mojap-hub-exports/",
```
Comment (jhpyke) [inline terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf]: From looking through the code - `bold_sm_spells` is calling `unload` as a macro during their model code - those buckets will be needed too.

**Review by jhpyke — CHANGES_REQUESTED** [review]
Need to rescope access to only the actual folders within the buckets we need, not the buckets as a whole

File: terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf (hunk 2 above)
Comment (jhpyke) [reply terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf]: Actually scratch that, the project is paused.

File: terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf (hunk 1 above)
Comment (murad-ali-MoJ) [reply terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf]: change accordingly

File: terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf [hunk 3]
```diff
@@ -14,6 +14,10 @@ data "aws_iam_policy_document" "create_a_derived_table" {
       "arn:aws:s3:::mojap-derived-tables",
       "arn:aws:s3:::dbt-query-dump/*",
       "arn:aws:s3:::dbt-query-dump",
+      "arn:aws:s3:::mojap-manage-offences/ho-offence-codes/*",
+      "arn:aws:s3:::mojap-manage-offences/ho-offence-codes",
```
Comment (jhpyke) [inline terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf]: Hi sorry should have been clearer - the non-starred ones only refer to the `getBucket` powers, so should stay as was.

```suggestion
      "arn:aws:s3:::mojap-manage-offences/",
```

File: terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf [hunk 4]
```diff
@@ -14,6 +14,10 @@ data "aws_iam_policy_document" "create_a_derived_table" {
...
+      "arn:aws:s3:::mojap-hub-exports/probation_referrals_dump/*",
+      "arn:aws:s3:::mojap-hub-exports/probation_referrals_dump",
```
Comment (jhpyke) [inline terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf]: Same as above

**Review by jhpyke — APPROVED** [review]
lgtm

**Pull request merged**